### ISSUE TITLE
Switching preferred container for a site (analytics.google.com) triggers a feedback loop of the site re-opening between accounts

### ISSUE THREAD

**Issue by RheingoldRiver** [mozilla/multi-account-containers#2503]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue (I was already on the most recent version, my browser auto-updates)
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

1. I had a container preference for analytics.google.com ("work")
2. I changed that preference to the container for my OSS project ("Tsubaki")
3. I opened analytics.google.com

### Actual behavior

The website opens repeatedly in a feedback loop of work-Tsubaki-work-Tsubaki-work-Tsubaki until I close that **window** (not just tab).

### Expected behavior

The preference should be switched to Tsubaki and the tab opened only once.

### Additional informations

I don't want to attempt to reproduce this multiple more times because the result is quite frustrating to deal with so I am not sure if google analytics is specific to the problem because this page redirects you upon loading. For example, a quick loading page like https://darthsanddroids.net/ which I use as my homepage might not have this issue.

### Provide a copy of Troubleshooting Information page (optional)

```plain text
https://analytics.google.com/analytics/web/?pli=1#
```

**Comment by dannycolin** [comment]
Could you make sure that analytics.google.com has been removed from the work container. See https://github.com/mozilla/multi-account-containers/wiki/Frequently-asked-questions#remove-always-open-in-site-from-a-container

**Comment by RheingoldRiver** [comment]
Ahh, so there *is* a way to manage what sites open in what containers without going directly to the tab! Thanks a lot for this, I was trying to find something like this in the addons preferences (a textarea with a json or something) but I couldn't (as a separate issue, I might suggest exposing this setting in that way, if you want I could open a separate ticket for that suggestion).

And yes, you are right, it is listed in both locations. So I can delete it from one and then problem solved! After this, I reproed it once more and then fixed it in the same way.

I would still say this is a bug though because (a) this doesn't feel like it's working as intended at all; and (b) the setting is very hidden, I don't think many people know it exists.

Either way, thank you for helping me solve this!

**Comment by dannycolin** [comment]
> I would still say this is a bug though because (a) this doesn't feel like it's working as intended at all;

It should have been deleted from your Work container when you assigned it to your Tsubaki container. Seems like something went wrong.

> the setting is very hidden, I don't think many people know it exists.

Agreed. I'm only a volunteer but I talked with the developer about updating the documentation and reworking the onboarding (installation) process to make it easier for users to find help. Hopefully, that's something that'll be enhanced soon.

I'm going to close this as fixed for now but feel free to ask any follow-up questions. You can also do so on support.mozilla.org under the Firefox desktop product.

**Comment by RheingoldRiver** [comment]
> Seems like something went wrong.

I can repro the issue though, so I don't think this is a one-off thing that something went wrong, but a bug that exists (potentially specific to sites that have redirection steps, like GA does). Not sure it should be closed?

**Comment by dannycolin** [comment]
> I can repro the issue though, so I don't think this is a one-off thing that something went wrong

Sorry I misunderstood when you said it was fixed. 

This is a bit hard to debugging since I don't have an account for these websites. The best solution would be to make a recording of the step-to-reproduce.

**Comment by RheingoldRiver** [comment]
Hmmm, so now I'm trying to reproduce, but I can't reproduce the same issue - When I say "open in Work container" (currently "Tsubaki") it just switches from work to Tsubaki.

I don't think you need to have a google analytics account to mess with this; just a google/gmail account is fine, it'll let you access analytics with that, and you don't need to actually have it attached to any existing website data.
